codestodians/food-basket-api
# [Feat: #150403839] create test endpoint

#### What does this PR do?
- This PR creates a test endpoint.

#### Description of Task to be completed?
- This PR creates a test endpoint that can be used to ensure that the front end

#### How should this be manually tested?
- Run `python manage.py migrate` to migrate table for test model
- Run `gunicorn config.wsgi` or `python manage.py runserver`
- Test endpoint by creating a GET/POST on `localhost:port/testmodel/`. It takes a name parameter that is a string. Check the port your server runs on.

#### Any background context you want to provide?
- The test endpoint is required to ensure that the frontend can make a request to the api and get a response.

#### What are the relevant pivotal tracker stories?
- [#150403839](https://www.pivotaltracker.com/story/show/150403839)
